include/fc/log/logger.hpp

```diff
@@ -21,7 +25,7 @@ namespace fc
    class logger 
    {
       public:
-         static logger get( const fc::string& name = "default");
+         static logger get( const fc::string& name = DEFAULT_LOGGER );
```

Comment:
Also use `DEFAULT_LOGGER` in `logging_config::default_config`.

Reply:
Done.